deel/torchlip/utils/lconv_norm.py

```diff
@@ -24,24 +24,54 @@
 # rights reserved. DEEL is a research program operated by IVADO, IRT Saint Exupéry,
 # CRIAQ and ANITI - https://www.deel.ai/
 # =====================================================================================
-from typing import Tuple
+from typing import Tuple, Union
 
 import numpy as np
 import torch
 import torch.nn as nn
 import torch.nn.utils.parametrize as parametrize
 
 
+def compute_lconv_coef_1d(
+    kernel_size: Tuple[int],
+    input_shape: Tuple[int] = None,
+    strides: Tuple[int] = (1,),
+    padding_mode: str = "zeros",
+) -> float:
+    stride = strides[0]
+    k1 = kernel_size[0]
+
+    if (
+        (padding_mode in ["zeros", "same"])
```

Comment:
shouldn't be `["zeros", "circular"]` ? 
```suggestion
        (padding_mode in ["zeros", "circular"])
```

Reply:
accepted in f3f32bbf560b48d3e96af79e931d0951afc710db